### ISSUE TITLE
Node Version...

### ISSUE THREAD

**Issue by milewski** [alexreardon/raf-stub#32]
Is there any technical reason to lock it down to 6.7 ??

```
error raf-stub@1.0.3: The engine "node" is incompatible with this module. Expected version "^6.7.0"
```

current version

```
node -v
v7.3.0
```

**Comment by alexreardon** [comment]
I will update the versions asap

**Comment by alexreardon** [comment]
No reason to lock it down.

**Comment by alexreardon** [comment]
I am not sure if this is technically a `bug` so I removed the label. I have fixed the issue and published it to npm on the version `1.0.4`. You should be good to go @Milewski
